yodalib/decompilers/ghidra/interface.py

```diff
@@ -341,6 +379,34 @@ def fill_comment(self, addr, user=None, artifact=None, **kwargs):
     # Artifact API
     #
 
+    def struct(self, name) -> Optional[Struct]:
```

Comment:
This looks like it should not be here right?

Reply:
Fixing...